cluster-operator/src/test/resources/log4j2-test.properties

```diff
@@ -1,4 +1,4 @@
-name = COConfig
+name = ClusterOperatorConfig
```

Comment:
it was COConfig in the main one above. How are we naming the others, TOConfig? UOConfig? Even in the Helm2 templating we are using COConfig.

Reply:
Yes, the others use abbreviations.

Reply:
So I would be consistent across components and between main code and tests.